code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -120,7 +120,40 @@
 	w_class = WEIGHT_CLASS_HUGE
 	hitsound = 'sound/weapons/sear.ogg'
 	damtype = BURN
-	attack_verb = list("punched", "cross countered", "pummeled")
+	light_color = "#E09D37"
+	attack_verb = list("singed", "seared", "burnt", "scorched")
+	var/lit = TRUE
+
+/obj/item/nullrod/godhand/Initialize()
+	. = ..()
+	update_brightness()
+
+/obj/item/nullrod/godhand/proc/update_brightness()
+	set_light(2)
+
+/obj/item/nullrod/godhand/get_heat()
+	return 1500
+
+/obj/item/nullrod/godhand/attack(mob/living/carbon/M, mob/user)
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
+		return
+	if(!isliving(M))
+		return
+	M.IgniteMob()
+	if(!ismob(M))
+		return
```

Comment:
Given that you've already done isliving (which checks for `mob/living`) you probably don't need this check here